cmd/microcloud/ask.go

```diff
@@ -1077,10 +1077,22 @@ func (c *initConfig) askOVNNetwork(sh *service.Handler) error {
 		return errors.New("User aborted")
 	}
 
-	// Ask the user if they want OVN.
-	wantsOVN, err := c.asker.AskBool("Configure distributed networking?", true)
-	if err != nil {
-		return err
+	wantsOVN := false
+
+	// When adding a new member it's important to check whether or not the existing cluster already
+	// has distributed networking configured.
+	// If a user opts out of the configuration (e.g. insterface selection) for distributed networking, this can be considered an error.
```

Comment:
```suggestion
	// If a user opts out of the configuration (e.g. interface selection) for distributed networking, this can be considered an error.
```

This sounds like adding an uplink-less member to a cluster with distributed networking is unsupported.
Could there be a case for supporting this?

Reply:
I have dropped one line of the comment because with the proposed changes MicroCloud enters an implicit distributed networking setup if enforced by the existing cluster. It doesn't anymore ask to configure the network and jumps right into the follow up questions like interface selection.
Currently for MicroCloud we expect all members to have the uplink network. 

Feel free to raise a feature request in https://github.com/canonical/microcloud/issues.